packages/runtime-common/realm.ts

```diff
@@ -968,30 +1000,24 @@ export class Realm {
     request: Request,
     neededPermission: 'read' | 'write',
   ) {
-      let endpontsWithoutAuthNeeded: RouteTable<true> = new Map([
-        // authentication endpoint
-        [
-          SupportedMimeType.Session,
-          new Map([
-            ['POST' as Method, new Map([['/_session', true]])],
-          ]),
-        ],
-        // SSE endpoint
-        [
-          SupportedMimeType.EventStream,
-          new Map([
-            ['GET' as Method, new Map([['/_message', true]])],
-          ]),
-        ],
-        // serve a text/html endpoint
-        [
-          SupportedMimeType.HTML,
-          new Map([
-            ['GET' as Method, new Map([['/.*', true]])],
-          ]),
-        ],
-      ]);
-    
+    let endpontsWithoutAuthNeeded: RouteTable<true> = new Map([
```

Comment:
typo: should be `endpointsWithoutAuthNeeded`

Reply:
Done 👍